<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GfxPaint::NewBufferDialog</class>
 <widget class="QDialog" name="GfxPaint::NewBufferDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>239</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Buffer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0" columnstretch="0,0,0">
   <item row="5" column="0">
    <widget class="QLabel" name="imageSizeLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Size</string>
     </property>
     <property name="buddy">
      <cstring>imageWidthSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="formatLabel">
     <property name="text">
      <string>Format</string>
     </property>
     <property name="buddy">
      <cstring>componentTypeComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="pixelRatioLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Pixel Ratio</string>
     </property>
     <property name="buddy">
      <cstring>pixelWidthSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="indexedCheckBox">
     <property name="text">
      <string>Indexed</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="componentSizeComboBox"/>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="componentCountComboBox"/>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QComboBox" name="componentTypeComboBox"/>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="imageWidthSpinBox">
     <property name="toolTip">
      <string>Image Width</string>
     </property>
     <property name="suffix">
      <string>px</string>
     </property>
     <property name="minimum">
      <number>16</number>
     </property>
     <property name="maximum">
      <number>4096</number>
     </property>
     <property name="value">
      <number>256</number>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QSpinBox" name="imageHeightSpinBox">
     <property name="toolTip">
      <string>Image Height</string>
     </property>
     <property name="suffix">
      <string>px</string>
     </property>
     <property name="minimum">
      <number>16</number>
     </property>
     <property name="maximum">
      <number>4096</number>
     </property>
     <property name="value">
      <number>256</number>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDoubleSpinBox" name="pixelWidthSpinBox">
     <property name="toolTip">
      <string>Pixel Width</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>0.125000000000000</double>
     </property>
     <property name="maximum">
      <double>8.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.125000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QDoubleSpinBox" name="pixelHeightSpinBox">
     <property name="toolTip">
      <string>Pixel Height</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>0.125000000000000</double>
     </property>
     <property name="maximum">
      <double>8.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.125000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>componentTypeComboBox</tabstop>
  <tabstop>indexedCheckBox</tabstop>
  <tabstop>imageWidthSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GfxPaint::NewBufferDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GfxPaint::NewBufferDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
